src/Bicep.Core.UnitTests/Utils/CompilationHelper.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 using System;
```

Comment:
This class is where the important changes are.  Everything else is just using these.

If you're good with the changes, I'll do another PR with the other half of the code converted to use them (this only converts about half the code).